controllers/node.go

```diff
@@ -564,13 +564,20 @@ func createNode(w http.ResponseWriter, r *http.Request) {
 	data.Node.Server = servercfg.GetServer()
 	if !logic.HostExists(&data.Host) {
 		logic.CheckHostPorts(&data.Host)
-		if servercfg.GetBrokerType() == servercfg.EmqxBrokerType {
-			// create EMQX credentials for host if it doesn't exists
-			if err := mq.CreateEmqxUser(data.Host.ID.String(), data.Host.HostPass, false); err != nil {
-				logger.Log(0, "failed to add host credentials to EMQX: ", data.Host.ID.String(), err.Error())
-				return
-			}
+	}
+	if servercfg.GetBrokerType() == servercfg.EmqxBrokerType {
```

Comment:
this logic needs to be applied on handleHostRegister func in controllers/enrollmentkeys.go

Reply:
done